### PR TITLE
Merge gallery backend implementations

### PR DESCRIPTION
**Pull request by philangist**
Merge my local gallery implementation with the existing implementation in this repo

### CODE REVIEW

File: controller.go [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-	"github.com/mikeydub/go-gallery.git/utils"
-)
-
-type galleryController struct {
-	appConfig  applicationConfig
-	httpClient http.Client
-}
-
-// might need this in the future
-type authGrantResponse struct {
```
Comment (philangist) [inline controller.go:15]: Will add some version of this back in the auth flow PR

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.env
+.idea/gallery.iml
```
Comment (Robinnnnn) [inline .gitignore:2]: what's this

Comment (philangist) [reply .gitignore:2]: Intellij idea config file. Local IDE artifact

**Review by Robinnnnn — APPROVED** [review]
gorgeous

**Review by kaitoo1 — APPROVED** [review]

File: internal/db/storage.go [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+	external_url,
+	created_date,
+	creator_address,
+	contract_address,
+--	token_id,
+	hidden,
+	image_thumbnail_url,
+	image_preview_url
+FROM nfts
+WHERE user_id='%s'
```
Comment (philangist) [inline internal/db/storage.go:50]: sql injectable, should fix before deploying to prod

**Pull request merged**